### DISCUSSION TITLE
Next planned release

### DISCUSSION THREAD

**Discussion by dorheini** [rytilahti/python-miio#1720] [General]
Hello
Curious about next planned release date..
I see that ijai.vacuum.v3 support was added which i'm looking for a while now..

Thanks,

**Comment by rytilahti** [comment]
See #1665 - the library is currently in flux and there are a few things to do before 0.6 is ready to be released.

**Reply by alvesmaicon** [reply]
Can we expect a future support for ijai.vacuum.v18? I tried to update the sound pack and i got the unsupported version warning

https://home.miot-spec.com/spec/ijai.vacuum.v18

**Reply by dsimop** [reply]
Hello @rytilahti, I would like to ask you if the 0.6 will come soon. It seems that many users have `ijai` models that are still not supported. Thank you.

**Reply by rytilahti** [reply]
I have just released [0.6.0.dev0](https://github.com/rytilahti/python-miio/releases/tag/0.6.0.dev0) and I'm hoping to release 0.6.0 soon-ish, but if you are interested in having this in homeassistant core [it will take some more time and effort](https://github.com/rytilahti/python-miio/issues/1808#issuecomment-1996273325).
